README.md

```diff
@@ -25,12 +25,41 @@ Composer will install the library to your project's `vendor/goaop/parser-reflect
 Usage
 ------------
 
+### Initialization
+
+Prior to first use library needs can optionally be initialized.
+
+Following code will use Composer's autoloader of project, where library is used as a dependency:
+
+```php
+$locator = new \Go\ParserReflection\Locator\ComposerLocator();
+\Go\ParserReflection\ReflectionEngine::init($locator);
+```
+
+Following code will use Composer's autoloader located at a given path:
+
+```php
+$composerLoader = require_once '/path/to/vendor/autoload.php';
+$composerLoader->unregister();
+
+$locator = new \Go\ParserReflection\Locator\ComposerLocator($composerLoader);
+\Go\ParserReflection\ReflectionEngine::init($locator);
+```
+
+If project is using custom autoloader, then:
+
+1. create class, that implements `\Go\ParserReflection\LocatorInterface`
+2. use that class instance as loader during initialization
+
+### Actual Usage
```

Comment:
I've used `Actual` word here, because there is already `Usage` section above this one. I thought this might confuse reader to see 2 `Usage` sections one inside another.